pkg/controller/jaeger/jaeger_controller.go

```diff
@@ -222,3 +232,57 @@ func (r *ReconcileJaeger) apply(jaeger v1.Jaeger, str strategy.S) error {
 
 	return nil
 }
+
+//addFinalizer will add this attribute to the Jaeger CR
+func (r *ReconcileJaeger) addFinalizer(instance *v1.Jaeger) error {
+	if len(instance.GetFinalizers()) < 1 && instance.GetDeletionTimestamp() == nil {
+		log.Info("Adding Finalizer for the Jaeger")
```

Comment:
`instance` should have a logger, which adds the name+namespace as fields to the message automatically. We also typically name it `jaeger` instead of `instance`, except when it's ambiguous.